api/api_ethereum.go

```diff
@@ -1200,6 +1200,12 @@ func (args *EthTransactionArgs) ToMessage(globalGasCap uint64, baseFee *big.Int,
 	// Reject invalid combinations of pre- and post-1559 fee styles
 	if args.GasPrice != nil && (args.MaxFeePerGas != nil || args.MaxPriorityFeePerGas != nil) {
 		return nil, errors.New("both gasPrice and (maxFeePerGas or maxPriorityFeePerGas) specified")
+	} else if args.GasPrice == nil && args.MaxFeePerGas == nil {
+		return nil, errors.New("the one of gasPrice and maxFeePerGas is not specified")
```

Comment:
if-condition says that both of `args.GasPrise` and `args.MaxFeePerGas` are not specified.

Reply:
I moved the condition in case baseFee is Zero (before KIP-71 hardfork) 
1. After hardfork, both fields are omited by users, we can use baseFee field after hardfork.
2. So only before hardfork, we need to check the condition.